src/test_typing.py

```diff
@@ -625,6 +625,19 @@ class MM1(MutableMapping[str, str], collections_abc.MutableMapping):
             class MM2(collections_abc.MutableMapping, MutableMapping[str, str]):
                 pass
 
+    def test_orig_bases(self):
+        T = TypeVar('T')
+        class C(typing.Dict[str, T]): ...
+        self.assertEqual(C.__orig_bases__, (typing.Dict[str, T],))
+
+    def test_multi_subscr_base(self):
+        T = TypeVar('T')
+        U = TypeVar('U')
+        V = TypeVar('V')
+        # these should just work
```

Comment:
That's slang for "these shouldn't crash", but it would be pedagogical to show what the various new dunder attributes of C and D actually are.